src/plugins/data/server/index_patterns/deprications/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
```

Comment:
This directory needs to be renamed - `deprications` => `deprecations`

Reply:
nice catch